descriptors/ModuleDescriptor-template.json

```diff
@@ -40,7 +40,7 @@
     },
     {
       "id": "pick-slips",
-      "version": "0.1",
+      "version": "0.2",
```

Comment:
Is this change going to be synchronised with UI changes that work with these breaking changes?

Reply:
Yes
https://issues.folio.org/browse/UICIRC-416